packages/main/test/pages/HAlignTable.html

```diff
@@ -0,0 +1,50 @@
+<!-- playground-fold -->
```

Comment:
What I meant with demokit sample is an actual sample, that can be seen by users consuming the table.

These samples are located in the **packages/website/docs/_component_pages** and **packages/website/docs/_samples/main** folder. There you can find table samples (such as Basic or StickyHeader), that can then be seen on the website.

Feel free to reach out, if you need help here